src/Middleware/RequestThrottling/src/Internal/RequestThrottlingLog.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.RequestThrottling.Internal
+{
+    internal static class RequestThrottlingLog
```

Comment:
Unless this is used in other places, our pattern is generally to make this a nested class *inside* `RequestThrottlingMiddleware`